### PR TITLE
The FOP/FOM Programming Guide

### PR DESCRIPTION
**Pull request by VenkyOS**
Signed-off-by: Venkataraman Padmanabhan <venkataraman.padmanabhan@seagate.com>

### CODE REVIEW

File: doc/FOPFOM_Programming_Guide.rst [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+
+- u8
+
+- void
+
+A FOP can be declared as follows
+
+::
+
+ record {
```
Comment (huanghua78) [inline doc/FOPFOM_Programming_Guide.rst:35]: As I stated in other PR. We need to use the following format to insert code.
`
```C
sequence { 
         u32 f_count;
         u8 f_buf
 } m0_fop_io_buf;
```

`

Comment (huanghua78) [reply doc/FOPFOM_Programming_Guide.rst:35]: I was wrong. This is for .md
Please use:
.. code-block:: C

**Review by huanghua78 — APPROVED** [review]

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: doc/FOPFOM_Programming_Guide.rst (hunk 1 above)
Comment (VenkyOS) [reply doc/FOPFOM_Programming_Guide.rst:35]: Completed.

**Pull request merged**